client/src/components/billSummaryPage.jsx

```diff
@@ -43,25 +45,35 @@ class BillSummaryPage extends React.Component {
     this.setState({
       split: 'Evenly'
     });
+    this.props.setTopLevelState('splitMechanism', 'Evenly')
   }
 
   splitByItem () {
     this.setState({
       split: 'by Item'
     });
+    this.props.setTopLevelState('splitMechanism', 'by Item')
   }
 
   render () {
     return (
       <div>
         <h3>Final Bill Split {this.state.split}</h3>
-        <SplitList guests={this.state.guests} totalCost={this.getBillTotal(this.state.guests)} tipPercentage = {this.state.tipPercentage} split={this.state.split}/>
+        <SplitList guests={this.state.guests} totalCost={this.getBillTotalWithoutTipOrTax(this.state.guests)} tipPercentage = {this.state.tipPercentage} split={this.state.split} zipCode = {this.state.zipCode}/>
         <form>
           Tip Percentage (%):
-          <input type="number" id="tipPercentage" name="tipPercentage" min="0" max="1000" value={this.state.tipPercentage} onChange={this.addTip}/><br/>
+          <input type="number" id="tipPercentage" name="tipPercentage" min="0" max="1000" value={this.state.tipPercentage} onChange={this.changeTipPercentage}/><br/>
         </form>
-        <button onClick={this.splitEvenly}>Split Evenly</button>
-        <button onClick={this.splitByItem}>Split by Item</button>
+        <Link to = "/select-food">
+          <button>Back</button>
+        </Link>
+        {this.state.guests.length <= 1
+          ? <div>
+            <button onClick={this.splitEvenly}>Split Evenly</button>
+            <button onClick={this.splitByItem}>Split by Item</button>
+            </div>
+          : <div/>
```

Comment:
This is not valid HTML and won't render anything. If you want to render only if there is 2 or more guests, you can do this:
```suggestion
          : <></>
```